pkg/skaffold/docker/parse.go

```diff
@@ -63,7 +63,13 @@ var (
 	RetrieveImage = retrieveImage
 )
 
-func readCopyCmdsFromDockerfile(onlyLastImage bool, absDockerfilePath, workspace string, buildArgs map[string]*string, cfg Config) ([]fromTo, error) {
+func readCopyCmdsFromDockerfile(onlyLastImage bool, absDockerfilePath, workspace string, buildArgs map[string]*string, cfg Config) (sl []fromTo, err error) {
+	defer func() {
```

Comment:
I think the error is coming from https://github.com/GoogleContainerTools/skaffold/blob/d9cf47da6aa179cc9b47e433b386491120f49578/pkg/skaffold/docker/parse.go#L361 where skaffold dockerfile parser assumes the node next to `FORM` is not nil. 
It could be nil if from is empty. 

I feel the right fix is to check if the node.next is not nil?